nearlib/test/integration.test.js

```diff
@@ -1,18 +1,23 @@
-const { Account, SimpleKeyStoreSigner, InMemoryKeyStore, KeyPair, LocalNodeConnection, NearClient, Near } = require('../');
+const { Account, KeyPair, Near, InMemoryKeyStore } = require('../');
+//const { InMemoryKeystore } = require('../signing');
+const dev = require('../dev');
 const fs = require('fs');
+const aliceAccountName = "alice.near";
+let nearjs;
+let account;
+let keyStore;
 
-const aliceAccountName = 'alice.near';
-const aliceKey = new KeyPair(
-    '22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV',
-    '2wyRcSwSuHtRVmkMCGjPwnzZmQLeXLzLLyED1NDMt4BjnKgQL6tF85yBx6Jr26D2dUNeC716RBoTxntVHsegogYw'
-);
-const test_key_store = new InMemoryKeyStore();
-const simple_key_store_signer = new SimpleKeyStoreSigner(test_key_store);
-test_key_store.setKey(aliceAccountName, aliceKey);
-const localNodeConnection = new LocalNodeConnection('http://localhost:3030');
-const nearClient = new NearClient(simple_key_store_signer, localNodeConnection);
-const account = new Account(nearClient);
-const nearjs = new Near(nearClient);
+beforeAll(async () => {
+    const options = {};
+    options.nodeUrl = 'http://localhost:3030';
```

Comment:
not sure why not spell out full object literal with all the properties.

It's JS anti-pattern to assemble object one property in time when everything is already available. It's more code and though it doesn't matter here it hurts JIT efficiency, see here for more details: https://mathiasbynens.be/notes/shapes-ics